app-text/mupdf/mupdf-1.19.1.recipe

```diff
@@ -0,0 +1,110 @@
+graphics. It renders text with metrics and spacing accurate to within \
+fractions of a pixel for the highest fidelity in reproducing the look of a \
+printed page on screen.
+
+MuPDF is also small, fast, and yet complete. It supports PDF 1.7 with \
+transparency, encryption, hyperlinks, annotations, searching and more. It also \
+reads XPS/OpenXPS documents and CBZ (Comic Book archive) files."
+HOMEPAGE="https://www.mupdf.com/"
+COPYRIGHT="2006-2022 Artifex Software, Inc"
+LICENSE="AGPL-3.0"
+REVISION="1"
+SOURCE_URI="https://mupdf.com/downloads/archive/mupdf-$portVersion-source.tar.xz"
+CHECKSUM_SHA256="b5eac663fe74f33c430eda342f655cf41fa73d71610f0884768a856a82e3803e"
+SOURCE_DIR="mupdf-$portVersion-source"
+PATCHES="mupdf-$portVersion.patchset"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	mupdf$secondaryArchSuffix = $portVersion
+	lib:libmupdf$secondaryArchSuffix = $portVersion
+	cmd:muraster$commandSuffix
+	cmd:mutool$commandSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libjbig2dec$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	mupdf${secondaryArchSuffix}_devel = $portVersion
+	devel:libmupdf$secondaryArchSuffix
+	devel:libmupdfthird$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	mupdf$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	devel:libfreetype$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libjbig2dec$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libopenjp2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage mupdf$secondaryArchSuffix \
+	"$libDir"/libmupdf.so \
+	"$binDir"/mutool \
+	"$binDir"/muraster
+
+BUILD()
+{
+	# build shared library
+	make USE_SYSTEM_FREETYPE=yes USE_SYSTEM_HARFBUZZ=yes \
+		USE_SYSTEM_JBIG2DEC=yes USE_SYSTEM_JPEGXR=no \
+		USE_SYSTEM_LCMS2=no USE_SYSTEM_LIBJPEG=yes \
+		USE_SYSTEM_MUJS=no USE_SYSTEM_OPENJPEG=yes \
+		USE_SYSTEM_ZLIB=yes USE_SYSTEM_GLUT=yes \
+		USE_SYSTEM_CURL=yes USE_SYSTEM_GUMBO=no \
+		shared=yes build=release $jobArgs
+}
+
+INSTALL()
+{
+	make USE_SYSTEM_FREETYPE=yes USE_SYSTEM_HARFBUZZ=yes \
+		USE_SYSTEM_JBIG2DEC=yes USE_SYSTEM_JPEGXR=no \
+		USE_SYSTEM_LCMS2=no USE_SYSTEM_LIBJPEG=yes \
+		USE_SYSTEM_MUJS=no USE_SYSTEM_OPENJPEG=yes \
+		USE_SYSTEM_ZLIB=yes USE_SYSTEM_GLUT=yes \
+		USE_SYSTEM_CURL=yes USE_SYSTEM_GUMBO=no \
+		build=release shared=yes install \
+		prefix=$prefix \
+		bindir=$binDir \
```

Comment:
Since you're using $commandSuffix this should be:
```diff
- 		bindir=$binDir \
+ 		bindir=$commandBinDir \
```

Reply:
Thanks, I just carried this stuff over from the original recipe because I didn't really understand it. I'll make the changes and commit

Reply:
cmd's are placed in `$prefix/bin/x86` with `$binDir`, with `$commandBinDir` they are placed in `$prefix/bin `

Reply:
Ah, understood. Will you make this change as part of your work on the recipe or shall I?

Reply:
I'll make the suggestions, the rest is on you to change it in the recipe here. :)

Reply:
Works for me. New commit incoming shortly, cheers